releases/release-4.0.0-ga.md

```diff
@@ -0,0 +1,114 @@
+---
+title: TiDB 4.0.0 GA Release Notes
+category: Releases
+---
+
+# TiDB 4.0.0 GA Release Notes
+
+Release date: May 28, 2020
+
+TiDB version: 4.0.0 GA
+
+## Compatibility changes
+
+* TiDB
+    - Optimize the error message of the large-sized transactions for easier troubleshooting [#17219](https://github.com/pingcap/tidb/pull/17219)
+
+* TiCDC
+    - Optimize the structure of the `Changefeed` configuration file to improve usability [#588](https://github.com/pingcap/ticdc/pull/588)
+    - Add the `ignore-txn-start-ts` configuration item, and change the condition from `commit_ts` to `start_ts` during transactions filtering [#589](https://github.com/pingcap/ticdc/pull/589)
+
+## Important bug fixes
+
+* TiKV
+    - Fix the `DefaultNotFound` error that occurs when backing up using Backup & Restore (BR) [#7937](https://github.com/tikv/tikv/pull/7937)
+    - Fix system panics caused by the out-of-order `ReadIndex` packages  [#7930](https://github.com/tikv/tikv/pull/7930)
+    - Fix system panics caused by incorrectly removing snapshot files after TiKV is restarted [#7927](https://github.com/tikv/tikv/pull/7927)
+
+* TiFlash
+    - Fix the possible data loss issue that occurs when the system panics because of incorrect processing logic of `Raft Admin Command`
+
+## New features
+
+* TiDB
+    - Add the `committer-concurrency` configuration item to control the number of `goroutines` in the retry-commit phase [#16849](https://github.com/pingcap/tidb/pull/16849)
+    - Support the `show table partition regions` syntax [#17294](https://github.com/pingcap/tidb/pull/17294)
+    - Add the `tmp-storage-quota` configuration item to limit the temporary disk space used by the TiDB server [#15700](https://github.com/pingcap/tidb/pull/15700)
+    - Support checking whether the partition table uses a unique prefix index when creating and changing tables [#17213](https://github.com/pingcap/tidb/pull/17213)
```

Comment:
Suggestion from Lilian: either `partition table` or `partitioned table` is acceptable, but we prefer the latter.
```suggestion
    - Support checking whether the partitioned table uses a unique prefix index when creating and changing tables [#17213](https://github.com/pingcap/tidb/pull/17213)
```